.github/ISSUE_TEMPLATE/setup.yml

```
@@ -0,0 +1,30 @@
+---
+name: Setup
+about: Prepare for development with basic project setup.
+title: '[ prep ] '
+labels: ''
+assignees: ''
+
+---
+
+Please check off line-items as they are completed and leave notes if necessary.
+If an item is not relevant to this project, [strike it out](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)
+(e.g. `~~Not relevant item~~`) or remove it. If child tickets are created for
+any line-item, please update this ticket description to include references to
+those tickets.
+
+- [ ] Create a README.
+- [ ] Set up [Tugboat](https://dashboard.tugboat.qa/).
+- [ ] Set up Dependabot ([example](https://github.com/ChromaticHQ/chromatichq.com/blob/master/.github/dependabot.yml)).
+- [ ] Set up PHP code style checks.
+- [ ] Set up front-end linting.
+- [ ] Set up GitHub Actions (example)[https://github.com/ChromaticHQ/chromatichq.com/tree/master/.github/workflows]
+      to run code style checks on YML, shell scripts, and execute PHP
+      checks/front-end linting tools.
+- [ ] Set up `settings.php` file infrastructure.
+- [ ] Set up a `composer.json` file.
+- [ ] Set up front-end tooling.
```

Before front-end tooling is even added, we need to _have_ a theme. Likewise, it is important to build theme assets in remote environments, so we should add a line items for that as well.
```suggestion
- [ ] Create a theme at `web/themes/custom/THEME_NAME`.
- [ ] Set up front-end tooling. This should provide:
    - [ ] A task to build the theme’s static assets.
    - [ ] Development workflow tasks to build assets automatically, watch for changes, refresh the browser automatically, etc.
    - [ ] CSS and JS linting tasks, as well as any other testing tasks that relevant to the project.
- [ ] Document system-level requirements and step(s) to build theme assets in remote environments in the theme’s README.
- [ ] Update build scripts (used in Tugboat, production, and elsewhere) to install front-end dependencies and build theme assets.
- [ ] Update GitHub Actions to run any front-end linting and/or testing tasks.
```

Thoughts?